addon/styles/_colors.scss

```diff
@@ -27,22 +27,38 @@ $frost-color-green-5: #c2efc7 !default;
 $frost-color-teal-1: #099 !default;
 $frost-color-teal-2: #005957 !default;
 $frost-color-teal-3: #00d3cd !default;
-$frost-color-teal-4: #a1e7ff !default;
+$frost-color-teal-4: #9aefea !default;
 
-$frost-color-lilac-1: #8d6ccf !default;
+$frost-color-lilac-1: #a183db !default;
 $frost-color-lilac-2: #4c318c !default;
 $frost-color-lilac-3: #e8dffb !default;
-$frost-color-lilac-4: #c3aaee !default;
+$frost-color-lilac-4: #c3aaee !default;  // not in style guide
```

Comment:
This was a good catch! Can we make sure that UX knows and updates the Ciena Style Guide as well please.